src/csvcubed/utils/sparql_handler/csvw_state.py

```diff
@@ -25,6 +30,18 @@ class CsvWState:
     def __post_init__(self):
         self.primary_graph_uri = path_to_file_uri_for_rdflib(self.csvw_json_path)
 
+    """Private Functions"""
+
+    @cached_property
+    def column_definitions(self) -> Dict[str, List[ColumnDefinition]]:
```

Comment:
I'm not aware of any other queries which could make use of this. But it may well be worth considering if any changes you've been making could make use of this information.